helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -288,5 +288,5 @@ private fun Provenance.writeValueAsString(): String =
     sourceArtifact?.let {
         "url=${it.url}, hash=${it.hash.value}"
     } ?: vcsInfo?.let {
-        "type=${it.type}, url=${it.url}, path=${it.path}, revision=${it.revision}"
+        "type=${it.type}, url=${it.url}, path=${it.path}, revision=${it.resolvedRevision}"
```

Comment:
Commit message: s/Ouput/Output

Should the text then also say `resolvedRevision=`?

Reply:
Done.

I think just saying revision is fine because it is implied by the context that it is a resolved revision, and it is shorter. E.g. it shows provenance for scan results which is always the resolved revision. What do you think?

Reply:
I'm fine with whatever you decide here, I just wanted to ask / remind in case you wanted to change it but forgot it.